packages/scene-composer/src/common/entityModelConstants.ts

```diff
@@ -0,0 +1,38 @@
+import { KnownComponentType } from '../interfaces';
+
+// Scene Nodes
+const SCENE_COMPONENT_TYPE_ID_PREFIX = 'com.example.3d';
```

Comment:
Maybe we need to come-up with property variable names.

Reply:
property name constants? I'll add in each component's file in my next PR